### ISSUE TITLE
Unable to filter by siteId in gatsby

### ISSUE THREAD

**Issue by dan-hoerr** [craftcms/gatsby-source-craft#7]
### Description
I've added a second site to a fresh Craft 3.5.15.1 install for fr-CA localization. (siteId = 2) But so far I've been unable to query that from the gatsby side. I've tried manually deleting the `.cache` folder, running `gatsby clean`, and restarting both the PHP/Nginx servers as well as `gatsby develop`. 

Here's the query working within Craft's GraphiQL:

![craft-graphiql](https://user-images.githubusercontent.com/1398576/98420500-57116980-2055-11eb-91aa-3fa3a1dc06b6.PNG)

```
{
  entries(section: "blogPosts", siteId: "2") {
    title
    ... on blogPosts_blogPosts_Entry {
      richTextDefault
    }	
  }
}
```

and the query I tried in Gatsby's GraphiQL

![gatsby-graphiql](https://user-images.githubusercontent.com/1398576/98420516-61336800-2055-11eb-9865-bea8c197020b.PNG)
:

```
query MyQuery {
  allCraftBlogPostsBlogPostsEntry(filter: {siteId: {eq: 2}}) {
    nodes {
      title
      richTextDefault
    }
  }
}
```

### Steps to reproduce

1. Create a second site with a different locale
2. Set up your sections and fields to support the new locale
3. Create and enable an entry to be published under this locale
4. restart everything just to make sure.
5. try to query the new entry data from the gatsby side using a siteId filter

### Additional info

- Craft version: 3.5.15.1
- PHP version: PHP 7.3.24 (cli) (built: Nov  5 2020 22:49:15) ( NTS )
- Database driver & version: mysql 5.7
- Plugins & versions: 
-- Gatsby: only `gatsby-source-craft`
-- Craft:      
    "born05/craft-assetusage": "2.2.0.1",
    "craftcms/aws-s3": "1.2.11",
    "craftcms/cms": "^3.5.15.1",
    "craftcms/gatsby-helper": "1.0.0-beta.1",
    "craftcms/redactor": "2.8.3",
    "doublesecretagency/craft-inventory": "2.1.1",
    "fruitstudios/linkit": "1.1.12.1",
    "nystudio107/craft-retour": "3.1.42",
    "nystudio107/craft-seomatic": "3.3.22",
    "spicyweb/craft-fieldlabels": "1.3.1.2",
    "spicyweb/craft-neo": "2.8.13",
    "vlucas/phpdotenv": "^3.4.0",
    "yiisoft/yii2-redis": "~2.0.0"

**Comment by dan-hoerr** [comment]
The default site content does appear to be accessible:

![Capture](https://user-images.githubusercontent.com/1398576/98421415-4bbf3d80-2057-11eb-92ad-a0aab40346c2.PNG)

![Capture2](https://user-images.githubusercontent.com/1398576/98421421-4f52c480-2057-11eb-99db-ba7dfed86891.PNG)

**Comment by andris-sevcenko** [comment]
Thanks for reporting! When sourcing data, Gatsby does not provide ay site ID, so Gatsby data nodes end up with just the default site data in them.

**Comment by andris-sevcenko** [comment]
@dan-hoerr this is fixed as of 89db4153c8af134d2d6294c3e943af3f44b162ea and is documented [here](https://github.com/craftcms/gatsby-source-craft#sourcing-multiple-sites).

**Comment by dan-hoerr** [comment]
@andris-sevcenko this is great; everything is working as expected on my end now. Thanks so much!
